src/main/java/edu/harvard/iq/dataverse/dataaccess/ImageThumbConverter.java

```diff
@@ -673,7 +673,10 @@ public static String rescaleImage(BufferedImage fullSizeImage, int width, int he
         } catch (Exception ioex) {
             logger.warning("caught Exceptiopn trying to create rescaled image " + outputLocation);
             return null;
-        }
+        } finally
```

Comment:
line 228 gets reported in Eclipse as having a leak because the FileOutputStream used to create the Channel there is never closed.

Reply:
However, it looks like closing the Channel does close the stream, so I don't think the warning is valid...